Builds/CMake/deps/Lz4.cmake

```diff
@@ -75,5 +77,7 @@ endif()
 
 add_dependencies (lz4_lib lz4)
 target_link_libraries (ripple_libs INTERFACE lz4_lib)
-exclude_if_included (lz4)
+if (${lz4_project})
+  exclude_if_included (lz4)
+endif()
 exclude_if_included (lz4_lib)
```

Comment:
Shouldn't lz4_lib also be excluded only when it's added via ExternalProject_Add?

Reply:
`lz4_lib` is added on line 5 regardless. `exclude_if_included` may not be the best name - it basically keeps the library from building if the whole project is included in another project. So, no, it should always be checked for exclusion.